controllers/gitopsservice_controller.go

```diff
@@ -811,14 +815,26 @@ func newBackendService(ns types.NamespacedName) *corev1.Service {
 	return svc
 }
 
-func newNamespace(ns string) *corev1.Namespace {
+func newRestrictedNamespace(ns string) *corev1.Namespace {
 	objectMeta := metav1.ObjectMeta{
 		Name: ns,
 		Labels: map[string]string{
 			// Enable full-fledged support for integration with cluster monitoring.
 			"openshift.io/cluster-monitoring": "true",
 		},
 	}
+
+	if strings.HasPrefix(ns, "openshift-") {
+		// Set pod security policy, which is required for namespaces pre-fixed with openshift
+		// as the pod security label syncer doesn't set them on OCP namespaces.
+		objectMeta.Labels["pod-security.kubernetes.io/enforce"] = "restricted"
+		objectMeta.Labels["pod-security.kubernetes.io/enforce-version"] = "v1.29"
```

Comment:
Hi @ibihim, just trying to understand this change. Was there any special reason to set it to v1.29 and not latest?